lib/league_statistics.rb

```diff
@@ -53,63 +21,74 @@ def scores(side)
     end
     scores
   end
-  #
-  # def average_team_scores
-  #   average_scores(team_scores)
-  # end
-  #
-  # def average_visitor_scores
-  #   average_scores(visitor_scores)
-  # end
-  #
-  # def average_home_team_scores
-  #   average_scores(home_team_scores)
-  # end
-  #
-  # def average_scores(side_scores_hash)
-  #   average_scores = Hash.new
-  #   side_scores_hash.each do |team, scores|
-  #     average_scores[team] = (scores.sum.to_f / scores.count).round(2)
-  #   end
-  #   average_scores
-  # end
-  #
-  # def best_offense
-  #   highest_score(average_team_scores)
-  # end
-  #
-  # def highest_scoring_visitor
-  #   highest_score(average_visitor_scores)
-  # end
-  #
-  # def highest_scoring_home_team
-  #   highest_score(average_home_team_scores)
-  # end
-  #
-  # def highest_score(average_side_scores)
-  #   highest_score = average_side_scores.max_by do |team, av_score|
-  #     av_score
-  #   end[0]
-  #   team_name(highest_score)
-  # end
-  #
-  # def worst_offense
-  #
-  #   lowest_score(average_team_scores)
-  # end
-  #
-  # def lowest_scoring_visitor
-  #   lowest_score(average_visitor_scores)
-  # end
-  #
-  # def lowest_scoring_home_team
-  #   lowest_score(average_home_team_scores)
-  # end
-  #
-  # def lowest_score(average_side_scores)
-  #   lowest_score = average_side_scores.min_by do |team, av_score|
-  #     av_score
-  #   end[0]
-  #   team_name(lowest_score)
-  # end
+
```

Comment:
All of these methods still work as they call on other methods rather than calling on the csv files.